galata/test/documentation/debugger.test.ts

```diff
@@ -157,15 +157,48 @@ test.describe('Debugger', () => {
     // Don't wait as it will be blocked
     void page.notebook.runCell(1);
 
-    // Wait to be stopped on the breakpoint
+    // Wait to be stopped on the breakpoint and the local variables to be displayed
     await page.debugger.waitForCallStack();
+    await expect(page.locator('select[aria-label="Scope"]')).toHaveValue(
+      'Locals'
+    );
 
     expect(
       await page.screenshot({
         clip: { y: 58, x: 998, width: 280, height: 138 }
       })
     ).toMatchSnapshot('debugger_variables.png');
 
+    // Expect the context menu for local variables to match snapshot
+    const variable = await page
+      .locator('.jp-DebuggerVariables-body li:first-child')
+      .textContent();
+    await page.click('.jp-DebuggerVariables-body li:first-child', {
+      button: 'right'
+    });
+    expect(await page.locator('.lm-Menu-content').screenshot()).toMatchSnapshot(
+      'debugger_variables_context-locals.png'
+    );
```

Comment:
Could you test for text content rather than snapshot? It will make the test more robust and faster.

Reply:
Thanks, fixed in https://github.com/jupyterlab/jupyterlab/pull/13476/commits/abdd2cdd68c86bb20c3d126852722b490f515a5c